backend/globaleaks/handlers/rtip.py

```diff
@@ -399,7 +398,6 @@ def put(self, tip_id):
 
         # TODO A 202 is returned regardless of whether or not an update was performed.
         self.set_status(202)  # Updated
```

Comment:
So, in this function `RTipInstance.put` reachable via `/receiver/{{ tip_id }}` the PUT request could contain invalid options and fail within the put handler.

However, as it is written, if the operation equals something other than "set" or "postpone" the handler still sets the response to 202. Clearly, the PUT operation did nothing. Shouldn't there be an `else: self.set_status(40x)` clause that catches these cases.